e2e/user/user-can-edit-account.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect } from "@playwright/test";
+import { loginKnownUser } from "../_lib/pom/init/login-known-user";
+import { gotoUserMyAccountPage } from "../_lib/sequences/nav-gotos";
+import { UserMyAccountEditPage } from "../_lib/pom/pages/user-my-account-edit-page";
+
+test("user can login, go to edit page, but cancel, and should not see their details changed", async ({
+  page,
+}) => {
+  // GIVEN
+  const { context, knownUser } = await loginKnownUser({ page });
```

Comment:
For our simple test setup where every test runs agains the same database, it is better to use the registerTestUser^1 init function when testing edits/cancellations^2 so that we can avoid changing information pertaining to the Known User. Such changes would break other tests that expect the Known User to have certain values.

_^1 - registerTestUser creates a random user account and returns the generated userName, userEmail, userPhoneNumber.^3_

_^2 - Although the cancel test ought not to change any data, it may if it fails. Thus, using registerTestUser would provide better test isolation even in the cancel-case._

_^3 - Running `make local-accounts` will drop old data and recreate the known state. This can be used from time to time to remove the users created by registerTestUser._

Reply:
Addressed 76415d0b7a43453a12a8e9f100ff2ea5c5349abf